.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+| Q | A
+| --- | ---
+| Fixed tickets | fixes #issuenum
+| License | MIT
+
+#### What's in this PR?
+
+Explain the contents of the documentation change.
```

Comment:
I think it makes sense no? "Which topic does the PR cover?" is ambiguous and overlaps with the first section. "Which problem does the PR fix?", OTOH, clearly asks for reasons why a PR is needed.